test/index.html

```diff
@@ -145,6 +145,9 @@
       .blocklyKeyboardNavigation
         .blocklyWorkspace:has(.blocklyActiveFocus)
         .blocklyWorkspaceFocusRing,
+      .blocklyKeyboardNavigation
+        .blocklySvg:has(~ .blocklyBlockDragSurface .blocklyActiveFocus)
```

Comment:
seems fine if it works, but what does the `~` do when it's at the front like that? all the examples I found have it in between two other selectors, so I don't know what this is actually doing

Reply:
I just copied it from @microbit-matt-hillsdon's comment, but it appears to let you target the first part of the ~ relationship, i.e. an element with the blocklySvg class that has a sibling drag surface that has active focus. https://developer.mozilla.org/en-US/docs/Web/CSS/:has#with_the_sibling_combinator discusses this for +, but the idea is the same, just without the strict-next-sibling qualification.

Reply:
Thanks for the link!